src-docs/src/views/page/page_content_center_with_side_bar.js

```diff
@@ -16,7 +16,7 @@ import {
 export default () => (
   <EuiPage>
     <EuiPageSideBar>SideBar nav</EuiPageSideBar>
-    <EuiPageBody>
+    <EuiPageBody component="div">
```

Comment:
Hey EUI team, how worried should we be about implementing devs copying this part? 

(It needs to be a `div` here because it's all wrapped in a `main` tag already and we shouldn't have nested `main` tags... but we don't want implementing devs to be removing the `main` tag from their code...)

If we should be worried, would adding a proper snippet here mitigate the issue or do we need to think of something else?

Reply:
```suggestion
    {/* The EUI docs site  already has a wrapping <main> tag, so we've changed this example to a <div> for accessibility. You likely don't need to copy the `component` prop for your own usage. */}
    <EuiPageBody component="div">
```

I would just add a JSX comment in the code to explain why it is this way.